Add recommendation for isFullScreen parameter (#10186)
MicrosoftDocs/msteams-docs: msteams-platform/resources/schema/manifest-schema.md

>
> * `isFullScreen` only works for apps published to your organization. Uploaded and published third-party apps can't use this property (it's ignored).
>

> > * `isFullScreen=true` removes the Teams-provided header bar and title from personal apps and task module dialogs. Is not recommended to use this parameter on chat bot apps.

Reviewer: ```suggestion
The `isFullScreen=true` parameter eliminates the header bar and title provided by Teams from personal apps and dialogs. However, it's recommended not to use the `isFullScreen=true` parameter with chat bot apps.
```
